control_plane/src/compute.rs

```diff
@@ -324,6 +324,9 @@ impl PostgresNode {
         conf.append("shared_preload_libraries", "neon");
         conf.append_line("");
         conf.append("neon.pageserver_connstring", &pageserver_connstr);
+        if let AuthType::NeonJWT = auth_type {
+            conf.append("neon.safekeeper_token_env", "$ZENITH_AUTH_TOKEN");
```

Comment:
This won't be needed if we make $ZENITH_AUTH_TOKEN the default GUC value -- and better reuse it as a token to both  pageserver and safekeeper (can be in separate pr).

Reply:
Finally decided to keep `NULL` the default as the current default is no-auth.